libcudacxx/docs/extended_api/synchronization_primitives/barrier.md

```diff
@@ -60,6 +61,43 @@ Programs shall ensure that this transformation would not introduce errors, for
 Under CUDA Compute Capability 6 (Pascal) or prior, an object of type
   `cuda::barrier` or `cuda::std::barrier` may not be used.
 
+## Shared memory barriers with transaction count
```

Comment:
I think this section needs a diff on the wording for the condition of phase completion.

Reply:
This has now been added in the subsection phase completion.,